### ISSUE TITLE
Modifying default Password hash through LDAP_PASSWORD_HASH cause configuration issue

### ISSUE THREAD

**Issue by abmeddeb** [bitnami/containers#49672]
### Name and Version

bitnami/openldap 2.6.6

### What architecture are you using?

amd64

### What steps will reproduce the bug?

Add `LDAP_PASSWORD_HASH={CRYPT}` or whatever valid value and start a container

### What is the expected behavior?

In the configuration database, `olcPasswordHash` attribute must be set to the `{CRYPT}` value

### What do you see instead?

In the configuration database, the `olcPasswordHash` attribute is set to the `{CRYPT}` value, with two braces at the end. This causes desired hash not to work

**Comment by gongomgra** [comment]
Hi @abmeddeb,

Thank you for reporting this issue. I have reproduced it and I'm working on a fix. I will mark this ticket as "on-hold" to avoid the stale bot to automatically close it. We will keep you posted.

**Comment by gongomgra** [comment]
Hi @abmeddeb,

Just a quick note to let you know that we have released new images for OpenLDAP 2.5 (`2.5.16-debian-11-r55`) and 2.6 (`2.6.6-debian-11-r55`) fixing this error. 

To double-check it, I updated the `bitnami/openldap:2.6` image in my machine, and then added the `LDAP_PASSWORD_HASH={CRYPT}` environment variable to the `docker-compose.yml` file, and checked the `/opt/bitnami/openldap/share/password_hash.ldif` file once the container has been started. You can check the results below

```
$ grep -C5 LDAP_PASSWORD_HASH docker-compose.yml
    environment:
      - LDAP_ADMIN_USERNAME=admin
      - LDAP_ADMIN_PASSWORD=adminpassword
      - LDAP_USERS=user01,user02
      - LDAP_PASSWORDS=password1,password2
      - LDAP_PASSWORD_HASH={CRYPT}
    volumes:
      - 'openldap_data:/bitnami/openldap'

volumes:
  openldap_data:

$ docker-compose up -d
(...)
$ docker exec -it -u root 7aec4c30bf7b bash
root@7aec4c30bf7b:/# cat opt/bitnami/openldap/share/password_hash.ldif
dn: olcDatabase={-1}frontend,cn=config
changetype: modify
add: olcPasswordHash
olcPasswordHash: {CRYPT}
```

We are closing this ticket as solved. Please do not hesitate to reopen it in case you find any issues.

**Comment by abmeddeb** [comment]
Hi,
Well done. Thanks for the quick solve.
cheers
